app/backend/src/couchers/servicers/requests.py

```diff
@@ -636,6 +649,31 @@ def MarkLastSeenHostRequest(self, request, context, session):
         session.commit()
         return empty_pb2.Empty()
 
+    def SetHostRequestArchiveStatus(self, request, context, session):
+        host_request = session.execute(
+            select(HostRequest)
+            .where_users_column_visible(context, HostRequest.surfer_user_id)
+            .where_users_column_visible(context, HostRequest.host_user_id)
```

Comment:
Please drop these filters: we will have it so that hidden users will still show up in requests (e.g. if you receive a req from someone who gets banned), so it's good people can archive them.